### ISSUE TITLE
Framework support for MoleculerJS

### ISSUE THREAD

**Issue by icebob** [stackblitz/webcontainer-core#221]
**Describe the solution you'd like:**
Is it available to add [Moleculer-based](https://github.com/moleculerjs/moleculer) Node.js template project?

**Provide a link to a github repo containing an example of the framework:**
https://github.com/icebob/stackblitz-moleculer-project-demo/tree/master

Working blitz: https://stackblitz.com/edit/moleculer-project-demo

**Additional context:**

Thanks in advance!

**Comment by d3lm** [comment]
Hey there! I have looked at your project and there seem to be some issues but it's related to some internal bug. Pressing `CTRL+C` kills the process immediately. I suppose you need to press it twice, at least that's why it prints. So something is fishy. I ll see if I can investigate what's going on. Thanks creating the project! Super cool to see that it's still working pretty well tho, aside from the bug I ran into.

**Comment by icebob** [comment]
Yeah, it's awesome that a complete backend project can run in the browser. And it's blazing fast! It starts faster than on my local dev env :)

Yeah, I saw that something is not good, but it's happens randomly. Once it starts correctly, next time not. Thanks that you look into it.

**Comment by icebob** [comment]
@d3lm did you have time to check it?

**Comment by d3lm** [comment]
No unfortunately not yet. I will drop a comment here if I get around it. Hang in there a bit longer. Thanks!

**Comment by icebob** [comment]
Any news about it?

**Comment by d3lm** [comment]
No not yet. As I said, once we get around this we'll let you know. But you can always fork your own projects if you want to start a new molecule project. We are working out user templates but it's not ready yet.
